u2f-chrome-extension/jasmine/spec/SoftTokenHelperSpec.js

```diff
@@ -0,0 +1,175 @@
+describe("Spec testing", function() {
+  var pubKey, secKey;
+  var appIdHash, keyHandle, challengeHash, version, timeoutSeconds;
+  var enrollChallenges, signData;
+  var softTokenProfile, softTokenHelper;
+  var exampleRegistration;
+  var enrollHelperRequest, signHelperRequest;
+
+  beforeEach(function() {
+    // Define variables
+    pubKey = "041155B2F73ABECCD83986C65AAC0D0BA709762AF2194DA800851026BC828E39C069FF8B40F765CD6E36FF164CD31CB96EA7D4326A3C635B3BCDB6AA45E2FEF5CE";
+    secKey = "3C0D26EFA654915BAF619FD94E37084F4024CA3ED52438A2837D9ACB510051C0";
+    timeoutSeconds = 600;
+    appIdHash = "BaIGvIRlYOG5Dg8y8EKjFdr_4O22_4pcXXoZ7DSuiQk";
```

Comment:
similar to the keys, this is simply copied from existing challenges and re-used. I could change it to "appIdHash" for readability since it doesn't seem to be reused, but I wanted to make sure it didn't fail any checks.